src/components/feedback-form/docs.mdx

```diff
@@ -0,0 +1,67 @@
+---
+componentName: 'FeedbackForm'
+---
+
+The FeedbackForm component is used at the end of tutorials to collect user feedback.
+
+<LiveComponent>
+{`			<FeedbackForm
```

Comment:
i'm curious, why did we want to copy over this component instead of just updating and using it from react-components?

Reply:
Was it a styles thing?

Reply:
We could do that but it would affect other projects that are using the component if dependencies are updated

Reply:
Oh I think I misunderstood this question. Yeah the styles are different than in react-components